test/unit/mock/cmd/r-print-version

```diff
@@ -0,0 +1 @@
+[1] ‘0.5.1’
```

Comment:
Since you mention it in your code comments, can we add multiple lines here to match the edge case you refer to in order to ensure we're pulling in the version string you expect?